portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['My hair is blonde.', 'I am 6 feet 3 inches tall.', 'Soccer is my favorite sport.', 'I enjoy relaxing in my hammock on the beach.', 'The Los Angeles Lakers is my favorite sport team.', 'I go to school at UCSB.', 'I have lived in Los Angeles my entire life.', 'Io parlo italiano.'];
```

Comment:
I would actually suggest putting this in the HTML file itself instead of the JS (although I know that's what the template is doing).

I think it would be clearer to have each fact in a separate div and you can use CSS to hide/show whichever random fact.  WDYT?

Reply:
That's an interesting idea. I will create a new branch to work on this.

Reply:
Is this PR obsolete now?  I think PR https://github.com/rysimone/My-Portfolio/pull/3 handles most of the logic in this PR?

Reply:
Yea I think it is. You guys just wanted me to change the ids i had to classes and other minor fixes like that.